docs/resources/Projects/how-to-create-find-organize-projects.md

```diff
@@ -0,0 +1,195 @@
+---
+slug: how-to-create-find-organize-projects
+title: How to Create, Find, and Organize Projects
+tags: []
+toc_max_heading_level: 5
+---
+#
+## How to Create a Project
+To create a new project, go to the Dashboard and click the “Create New” button in the upper right corner. This will open the “Create Project” window where you can choose to start with a template app or a blank project. Name the project and then select a blank project or template. 
+
+![create-new-project.avif](..%2F..%2F..%2Fstatic%2Fimg%2Fcreate-new-project.avif)
+
+During project creation, you can access a three-step "Project Setup" window, which is optional. If you prefer to bypass this, simply click "Skip." All setup options will remain accessible under “Settings and Integrations” → “Firebase” → Wizard hat icon.
+
+## How to Find Projects
+Navigate to the Project Dashboard to view all your projects. You can search for specific projects using the search bar. 
+
+<div style={{
+    position: 'relative',
+    paddingBottom: 'calc(56.67989417989418% + 41px)', // Ensures the aspect ratio plus additional padding
+    height: 0,
+    width: '100%'
+}}>
+    <iframe 
+        src="https://demo.arcade.software/KWGdRdptt18rissIt8SQ?embed&show_copy_link=true"
+        title="Projects - FlutterFlow"
+        style={{
+            position: 'absolute',
+            top: 0,
+            left: 0,
+            width: '100%',
+            height: '100%',
+            colorScheme: 'light'
+        }}
+        frameBorder="0"
+        loading="lazy"
+        webkitAllowFullScreen
+        mozAllowFullScreen
+        allowFullScreen
+        allow="clipboard-write">
+    </iframe>
+</div>
+
+
+Narrow your search scope with the dropdown menu next to the search bar:
+
+- **My Private Projects:** Projects accessible only to you.
```

Comment:
Made a bold styling change here: 
Earlier it was My Private Projects: Projects accessible only to you.
Changed to **My Private Projects:** Projects accessible only to you. so titles look apart from title desc.